cdk/lib/support-workers.ts

```diff
@@ -232,5 +242,304 @@ export class SupportWorkers extends GuStack {
       logicalId: `SupportWorkers${this.stage}`,
       reason: "Moving existing step function to CDK",
     });
+
+    // Alarms
```

Comment:
would it make sense to split it out into several files rather than relying on comments? or is that not the done thing for CDK?

Reply:
I think it would make it more difficult to get a full picture of what's going on